docs/src/main/sphinx/release/release-459.md

```diff
@@ -0,0 +1,85 @@
+# Release 459 (25 Sep 2024)
+
+## General
+
+* Improve error reporting on input validation for the `approx_percentile`
+  function. ({issue}`23546`)
+* Fix possible query failure when `retry_policy` is set to `TASK` and when
+  adaptive join reordering is enabled. ({issue}`23407`)
+
+## Docker image
+
+* Update Java runtime to Java 23. ({issue}`23482`)
+
+## CLI
+
+* Display data sizes and rates with binary (1024-based) abbreviations such as
+  `KiB` and `MiB`. Add flag `--decimal-data-size` to use decimal (1000-based)
+  values and abbreviations such as `KB` and `MB`. (#13054)
+
+## BigQuery connector
+
+* Add support for dereference pushdown. ({issue}`23443`)
+* Fix query failure when the `bigquery.service-cache-ttl` property isn't `0ms`
+  and case insensitive name matching is enabled. ({issue}`23481`)
+
+## ClickHouse connector
+
+* Unconditionally pushdown `varchar` predicates. ({issue}`23516`)
+
+## Delta Lake connector
+
+* Allow configuring maximum HTTP requests in
```

Comment:
What does "maximum HTTP requests" mean? Is it referring to "concurrent HTTP requests", "connections", something else?

Reply:
In the docs we added 

 Maximum [integer](prop-type-integer) number of concurrent HTTP requests to
    Azure on every node. Defaults to double the number of processors on the
    node. Minimum `1`. Use this property to reduce the number of connections
    when you observe too many connections from your Trino cluster nodes to
    Azure Storage and encounter rate limiting issues.

I adjusted a bit towards that..